tools/stress/src/org/apache/cassandra/stress/settings/SettingsLog.java

```diff
@@ -93,11 +95,12 @@ public static final class Options extends GroupedOptions
         final OptionSimple hdrOutputFile = new OptionSimple("hdrfile=", ".*", null, "Log to a file", false);
         final OptionSimple interval = new OptionSimple("interval=", "[0-9]+(ms|s|)", "1s", "Log progress every <value> seconds or milliseconds", false);
         final OptionSimple level = new OptionSimple("level=", "(minimal|normal|verbose)", "normal", "Logging level (minimal, normal or verbose)", false);
+        final OptionSimple printThreadDumpOnShutdown = new OptionSimple("print-thread-dump-on-shutdown", "", null, "Print local thread dump to the standard output on shutdown (normal, SIGTERM, SIGINT, SIGABRT).", false);
```

Comment:
I will require additional code on dtest side to understand whethere given c-s support this option or not.
It would be better to just do that by default, but only if `c-s` was terminated.

Reply:
Reworked to a version without new option.